app/assets/javascripts/controllers/middleware_jdbc_driver/middleware_jdbc_driver_controller.js

```diff
@@ -29,10 +29,10 @@ function MwAddJdbcDriverController($scope, $http, miqService) {
     })
       .then(
         function(result) { // success
-          miqService.miqFlash(result.data.status, result.data.msg);
+          miqService.replacePartials(result.data);
         },
         function() { // error
-          miqService.miqFlash('error', 'Unable to add JDBC Driver "' + data.driverName + '" on this server.');
+          miqService.replacePartials(result.data);
```

Comment:
@rubenvp8510 how does it know whether to show success or an error flash message.